validator/keymanager/local/import.go

```diff
@@ -22,13 +22,6 @@ func (km *Keymanager) ImportKeystores(
 	keystores []*keymanager.Keystore,
 	passwords []string,
 ) ([]*ethpbservice.ImportedKeystoreStatus, error) {
-	if len(passwords) == 0 {
-		return nil, errors.New("no passwords provided for keystores")
-	}
-	if len(passwords) != len(keystores) {
-		return nil, errors.New("number of passwords does not match number of keystores")
-	}
```

Comment:
we can do this too if you think we need an extra layer of protection, the case for mismatched passwords is taken care of from the callers for this function in` ImportAccounts(ctx context.Context, cfg *ImportAccountsConfig) ([]*ethpbservice.ImportedKeystoreStatus, error)` as well as
```
 func (s *Server) ImportKeystores(
	ctx context.Context, req *ethpbservice.ImportKeystoresRequest,
) (*ethpbservice.ImportKeystoresResponse, error)
``` 
we could also do it that way if it is preferred.

Reply:
Thinking about this more, let's not delete this logic, but instead follow the approach suggested in my first comment. Removing these checks feels dangerous, as this exported function could be used by other code in the future that might not know the checks were removed.

Reply:
I added the check back in with the specific errors, I don't actually need to use the switch statements since I have that logic to trim or padd the password to be equal to the keystores.